spec/presenters/decidim/admin_log/organization_presenter_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::AdminLog::OrganizationPresenter, type: :helper do
+  include_examples "present admin log entry" do
+    let(:admin_log_resource) { organization }
+    let(:action) { "update_id_documents_config" }
+  end
+
+  include_examples "present admin log entry" do
+    let(:admin_log_resource) { organization }
+    let(:action) { "update_extra_user_fields" }
+  end
+
+  include_examples "present admin log entry" do
+    let(:admin_log_resource) { organization }
+    let(:action) { "update" }
+  end
+
+  describe "when the action is update_extra_user_fields" do
+    let(:action) { "update_extra_user_fields" }
+    let(:extra_user_fields) { true }
+
+    it "renders the extra user fields custom message" do
```

Comment:
If we extend the `OrganizationPresenter` we should just add specs on what we introduce to keep it dry